tests/basic.rs

```diff
@@ -1,11 +1,22 @@
-use std::time::SystemTime;
+extern crate fnv;
+extern crate portus;
+#[macro_use]
+extern crate slog;
+extern crate failure;
+extern crate libccp;
+extern crate minion;
+extern crate slog_term;
+extern crate time;
 
 use fnv::FnvHashMap as HashMap;
 use portus::lang::Scope;
 use portus::{DatapathTrait, Report};
-use slog;
+use std::time::Instant;
 
-use super::{IntegrationTest, ACKED_PRIMITIVE};
+mod libccp_integration;
+use libccp_integration::IntegrationTest;
+
+pub const ACKED_PRIMITIVE: u32 = 5; // libccp uses this same value for acked_bytes
```

Comment:
what is this?

Reply:
matches this: https://github.com/ccp-project/portus/pull/74/files#diff-e9ab6ea66183d998c731b0cb4ade8fc7R85

so that we can figure out what we expect the values to be.